profiles/ShanmukhiKairuppala.json

```
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/109336734/209447975-84bfd79f-1bcc-4cb8-8ff3-af312e98dcb7.jpg",
+    "issueId": 1586,
+    "name": "Shanmukhi Kairuppala",
+    "username": "ShanmukhiKairuppala"
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

Edited my PR by adding new line.